docs/develop/01_sdk/02_cookbook/04_claiming/05_presentation_verification.md

```diff
@@ -17,7 +17,8 @@ This means that the logic for Verifiers does not change depending on the case, t
 
 :::warning
 Verifying a presentation provides proof that all the information is correct and authentic, and that the credential has not been revoked.
-Verifiers still need to match the subject of the credential to the entity that is presenting it.
-One way of achieving this is by asking the Claimer to include a challenge in the presentation signature, as shown in the snippet above.
+Verifiers still need to 1. match the subject of the credential to the entity that is presenting it, and 2. make a judgment about the credential attester.
+For the former, one way of achieving this is by asking the Claimer to include a challenge in the presentation signature, as shown in the snippet above.
 Without a challenge, Verifiers must implement other measures to be certain about the identity of the presenter.
+For the latter, Verifiers must have a registry of attesters they trust, and verify that the issuer of the credential they are verifying belongs to such list and, where necessary, whether it is still in operation or not, i.e., whether its DID still exists or has been deleted.
 :::
```

Comment:
I think it's more readable to have a single warning inside such a block. We also should add captions to the block to give a hint about what the warning says.

```
:::warning Be mindful about which attester you trust!
...
:::

:::warning Ensure that the attester is still operating
...
:::

```